collect_app/src/main/java/org/odk/collect/android/forms/DatabaseFormsRepository.java

```diff
@@ -34,6 +37,22 @@ public boolean contains(String jrFormId) {
         }
     }
 
+    @Nullable
+    @Override
+    public Form get(Long id) {
+        try (Cursor cursor = new FormsDao().getFormsCursor(_ID + "=?", new String[]{id.toString()})) {
+            return getFormOrNull(cursor);
+        }
+    }
+
+    @Nullable
+    @Override
+    public Form get(String jrFormId, String jrVersion) {
+        try (Cursor cursor = new FormsDao().getFormsCursor(JR_FORM_ID + "=? AND " + JR_VERSION + "=?", new String[]{jrFormId, jrVersion})) {
```

Comment:
Totally missed this was a thing. On one hand it feels like it might make sense to start reducing the number of methods on `FormsDao` and just write queries as much as possible in our new repos. I'll see how used the `getFormsCursor(String formId, String formVersion)` is and think about inlining it.

Reply:
Actually going to avoid that method as it adds sorting in which we don't need here.